src/PlatformEngines.jl

```diff
@@ -29,7 +29,7 @@ This option is mainyl used to exclude symlinks from extraction (see: `copyderef`
 This method is initialized by `probe_platform_engines()`, which should be
 automatically called upon first import of `BinaryProvider`.
 """
-gen_unpack_cmd = (tarball_path::AbstractString, out_path::AbstractString; excludelist::Union{AbstractString, Cmd} = ``) ->
+gen_unpack_cmd = (tarball_path::AbstractString, out_path::AbstractString; excludelist::AbstractString = nothing) ->
```

Comment:
Same here

Reply:
Stupid me! Thanks for your vigilance and patience - it's my first PR ;-)

Reply:
You're doing a great job!  Keep up the good work!